Core/FigureData/IFigureDataManager.cs

```diff
@@ -11,4 +12,11 @@ public interface IFigureDataManager
     Palette GetPalette(int colorId);
     bool TryGetPalette(int palletId, out Palette palette);
     int GetRandomColor(int palletId);
+    private const string DefaultFigure =
+        "sh-3338-93.ea-1406-62.hr-831-49.ha-3331-92.hd-180-7.ch-3334-93-1408.lg-3337-92.ca-1813-62";
+
+    public static string FilterFigure(string figure)
```

Comment:
Personally not a fan of static methods on interfaces. Perhaps make this non static and move the implementation to the class?

Reply:
Yeah, i dont like so much as well haha

Fixing